bin/run_xgettext.sh

```diff
@@ -74,38 +86,39 @@ do
 	# On Windows, find still outputs the name of pruned folders
 	if [ ! -d "$file" ]
 	then
+		# shellcheck disable=SC2086  # KEYWORDS is meant to be splitted
 		xgettext $KEYWORDS -j -o "$OUTFILE" --from-code=UTF-8 "$file" || exit 1
-		sed -i "s/CHARSET/UTF-8/g" "$OUTFILE"
+		sed -i.bkp "s/CHARSET/UTF-8/g" "$OUTFILE"
```

Comment:
What's the `.bkp` for?

Reply:
`sed -i [suffix]`  edit files in place and makes backup if extension supplied, but on osx the suffix is required.
I forgot to remove the backup at the end....